src/main/java/service/Reader.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import java.util.List;
+
+public interface Reader {
+    List<String> fileReader(String filePath);
```

Comment:
```suggestion
    List<String> read(String filePath);
```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names